app/main/views/direct_award.py

```diff
@@ -0,0 +1,190 @@
+from flask import jsonify, abort, current_app, request
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy import asc, desc
+
+from dmapiclient.audit import AuditTypes
+from dmutils.config import convert_to_boolean
+from .. import main
+from ... import db
+from ...models import User, AuditEvent
+from app.models.direct_award import Project, Search
+from ...utils import (
+    get_json_from_request, get_int_or_400, json_has_required_keys, pagination_links,
+    get_valid_page_or_1, validate_and_return_updater_request)
+
+
+@main.route('/direct-award/projects', methods=['GET'])
+def list_projects():
+    page = get_valid_page_or_1()
+
+    projects = Project.query
+    user_id = get_int_or_400(request.args, 'user-id')
```

Comment:
I think we all find it confusing but it's 'consistent' with what we already have.